cartoframes/context.py

```diff
@@ -1021,62 +1138,99 @@ def data_augment(self, table_name, metadata):
-                  will return the original value, and 'denominated' will
-                  normalize by a denominator.
-                - `denom_id` (str, optional): Measure ID from DO catalog
-                - `numer_timespan` (str, optional): The desired timespan for
-                  the measurement. Defaults to most recent timespan available
-                  if left unspecified.
-                - `geom_timespan` (str, optional): The desired timespan for the
-                  geometry. Defaults to timespan matching `numer_timespan` if
-                  left unspecified.
-                - `target_area` (str, optional): Instead of aiming to have
-                  `target_geoms` in the area of the geometry passed as extent,
-                  fill this area. Unit is square degrees WGS84. Set this to
-                  `0` if you want to use the smallest source geometry for this
-                  element of metadata, for example if you're passing in points.
-                - `target_geoms` (str, optional): Override global
-                  `target_geoms` for this element of metadata
-                - `max_timespan_rank` (str, optional): Override global
-                  `max_timespan_rank` for this element of metadata
-                - `max_score_rank` (str, optional): Override global
-                  `max_score_rank` for this element of metadata
+            metadata (pandas.DataFrame): List of all measures to add to
+                `table_name`. See `CartoContext.data_discovery` outputs
+                for a full list of metadata columns.
+            persist_as (str, optional): Output the results of augmenting
+                `table_name` to `persist_as` as a persistent table on CARTO.
+                Defaults to ``None``, which will not create a table.
 
         Returns:
             pandas.DataFrame: A DataFrame representation of `table_name` which
             has new columns for each measure in `metadata`.
-        """
 
-        try:
-            with open(os.path.join(os.path.dirname(__file__),
-                                   'assets/data_obs_augment.sql'), 'r') as f:
-                augment_functions = f.read()
-            self.sql_client.send(augment_functions, do_post=False)
-        except Exception as err:
-            raise CartoException("Could not install `obs_augment_table` onto "
-                                 "user account ({})".format(err))
-
-        # augment with data observatory metadata
-        augment_query = '''
-            select obs_augment_table('{username}.{tablename}',
-                                     '{cols_meta}');
-        '''.format(username=self.creds.username(),
-                   tablename=table_name,
-                   cols_meta=json.dumps(metadata))
-        self.sql_client.send(augment_query, **DEFAULT_SQL_ARGS)
-
-        # read full augmented table
-        return self.read(table_name)
+        Raises:
```

Comment:
Maybe change to "If any column names in `table_name` match any value in the ``suggested_name`` column of `metadata`.